sys-auth/seatd/seatd-0.5.0.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 2020-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit meson systemd
+
+DESCRIPTION="Minimal seat management daemon and universal library"
+HOMEPAGE="https://sr.ht/~kennylevinsen/seatd"
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://git.sr.ht/~kennylevinsen/seatd"
+else
+	KEYWORDS="~amd64 ~arm64 ~ppc64 ~x86"
+	SRC_URI="https://git.sr.ht/~kennylevinsen/seatd/archive/${PV}.tar.gz -> ${P}.tar.gz"
+fi
+LICENSE="MIT"
+SLOT="0/1"
+IUSE="elogind systemd"
+REQUIRED_USE="?? ( elogind systemd )"
+
+DEPEND="
+	elogind? ( sys-auth/elogind )
+	systemd? ( sys-apps/systemd )
+"
+RDEPEND="${DEPEND}"
+BDEPEND=">=app-text/scdoc-1.9.7"
+
+src_configure() {
+	local emesonargs=(
+		-Dman-pages=enabled
+		-Dwerror=false
+	)
+
+	if use elogind || use systemd; then
+		emesonargs+=( -Dlogind=enabled )
+	else
+		emesonargs+=( -Dlogind=disabled )
+	fi
```

Comment:
```
option('seatd', type: 'feature', value: 'enabled', description: 'seatd support')
option('builtin', type: 'feature', value: 'disabled', description: 'builtin seatd server')
option('server', type: 'feature', value: 'enabled', description: 'seatd server')
```
Any sense to make any of these toggleable for the user?

Reply:
I don't think they make sense in gentoo, IIRC it's something that will depend from one distro/vendor (ie. static-linking) to another.